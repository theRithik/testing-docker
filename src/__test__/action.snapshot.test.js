import React from "react";
import  courseList from '../action/actionFile'
import {create} from 'react-test-renderer'

describe('Snapshot Test',()=>{
    test('testing courseList',()=>{
        let tree = create(<courseList/>);
        expect(tree.toJSON()).toMatchSnapshot()
    })
})