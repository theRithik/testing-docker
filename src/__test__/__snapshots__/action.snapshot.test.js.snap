// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test testing courseList 1`] = `<courseList />`;
